apps/users-service/src/framework/lambda.ts

```diff
@@ -28,6 +28,17 @@ export interface RequestContext {
   connection: MongoClient;
 }
 
+export const response = (res: APIGatewayProxyResult): APIGatewayProxyResult => {
+  return {
+    ...res,
+    headers: {
+      'Access-Control-Allow-Origin': '*',
```

Comment:
According to [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Origin) the wildcard does not work with credentials (and it'd also be insecure), so I think we'll need to pass in the `asdfghjkl.asap.yld.io` domain via the environment.